jmeter-service/jmeter/jmeter.conf.yaml

```diff
@@ -3,13 +3,25 @@ spec_version: '0.1.0'
 workloads:
   - teststrategy: performance
     vuser: 100
-    loopcount: 10
-    script: jmeter/load.jmx
+    loopcount: 500
+    script: jmeter/basicload.jmx
```

Comment:
Will this break our end-to-end tutorials? We expect the filename to be load.jmx and basiccheck.jmx
```
keptn add-resource --project=sockshop --stage=dev --service=carts --resource=jmeter/basiccheck.jmx --resourceUri=jmeter/basiccheck.jmx
keptn add-resource --project=sockshop --stage=staging --service=carts --resource=jmeter/load.jmx --resourceUri=jmeter/load.jmx
```
In addition, the higher `loopcount` means that the load test is executed more often, therefore our tutorial will take longer?

Reply:
this conf file - AFAIK - is not used but just shown as a reference. All our examples bring their own. But - I agree. I will revert it to what it was before